contributing.md

```diff
@@ -23,7 +23,7 @@ This document describes how to go about it. If you have any questions, get in to
 
 ## License
 
-By contributing your code, you agree to license your contribution under the terms of the [APLv2](https://www.apache.org/licenses/LICENSE-2.0).
+By contributing your code, you agree to license your contribution under the terms of the [APLv2](/LICENSE).
```

Comment:
I wonder if we should include the full link to the GitHub file, otherwise people viewing this file outside GitHub will get lost.
Also, should we rename the link to 'our license' or something?